src/coreComponents/codingUtilities/tests/testTable.cpp

```diff
@@ -0,0 +1,393 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2018-2020 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2020 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2018-2020 TotalEnergies
+ * Copyright (c) 2019-     GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+// Source includes
+#include "../Table.hpp"
+#include "../../dataRepository/Group.hpp"
```

Comment:
```suggestion
#include "codingUtilities/Table.hpp"
#include "dataRepository/Group.hpp"
```
Why do you have to `..` the paths?
I would expect that (e.g.) you could call the files directly using `codingUtilities/Table.hpp` since you're depending on `codingUtilities`? (But I'm not expert of `cmake` so I'm far from being sure.)